### PR TITLE
Replaced many strings with constants

### PR DESCRIPTION
**Pull request by danielkim10**
As the project grows and more prefabs are created, it will be helpful to rely on constants instead of raw strings.

Applied this to all scenes, sprites, objects, prefabs, tags, player pref keys, buttons, overlays

Did not apply this to: ingame strings

That will be for another PR...

### CODE REVIEW

File: Assets/Constants/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    public static readonly string SPRITE_OWNED_KEY = "owned";
+
+    // INVENTORY
+    public static readonly string SCROLL_CONTENT_INVENTORY = "ScrollContent";
+    public static readonly string SCROLL_VIEW_INVENTORY = "ScrollView";
+
+    // SHOP
+    public static readonly string BUY_BUTTON_SHOP = "BuyButton";
+    public static readonly string BUY_TEXT_SHOP = "BuyText";
+}
```
Comment (kcchik) [inline Assets/Constants/Constants.cs:56]: add newline

**Review by kcchik — CHANGES_REQUESTED** [review]
nice

File: Assets/Constants/Constants.cs (hunk 1 above)
Comment (danielkim10) [reply Assets/Constants/Constants.cs:57]: i hate you

**Review by Agam642 — APPROVED** [review]
Looks good just some string changes

**Review by gerathas — COMMENTED** [review]
Looks good

**Review by gerathas — APPROVED** [review]

**Review by Raymond-Xia — APPROVED** [review]
Looks swag, this guy actually has good code practices kekw

File: Assets/Constants/Constants.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    // INVENTORY
+    public static readonly string SCROLL_CONTENT_INVENTORY = "ScrollContent";
+    public static readonly string SCROLL_VIEW_INVENTORY = "ScrollView";
+
+    // SHOP
+    public static readonly string BUY_BUTTON_SHOP = "BuyButton";
+    public static readonly string BUY_TEXT_SHOP = "BuyText";
+    
+}
```
Comment (kcchik) [inline Assets/Constants/Constants.cs:57]: looks like its still missing a newline

**Review by kcchik — CHANGES_REQUESTED** [review]

File: Assets/Constants/Constants.cs (hunk 2 above)
Comment (kcchik) [reply Assets/Constants/Constants.cs:57]: mb

**Review by kcchik — APPROVED** [review]

**Pull request merged**